api/replication.storage/v1alpha1/volumereplication_types.go

```diff
@@ -36,6 +36,25 @@ const (
 	ConditionValidated = "Validated"
 )
 
+// These are valid messages for various
+// conditions and states of Volume
+// replication.
+const (
+	MessagePromoted               = "volume is promoted"
+	MessageHealthy                = "volume is healthy"
+	MessageNotResyncing           = "volume is not resyncing"
+	MessagePromotionErrorDetected = "error detected while promotion"
+	MessageDemotionErrorDetected  = "error detected while demotion"
+	MessageResyncErrorDetected    = "error detected while resyncing"
+	MessageValidated              = "volume is validated"
+	MessageFailedPromoted         = "failed to promote volume"
+	MessageFailedPreCondition     = "failed to meet prerequisite"
+	MessageDemoted                = "volume is demoted"
+	MessageDegraded               = "volume is degraded"
+	MessageResyncTriggered        = "resync is triggered"
```

Comment:
rsync was triggered? a more verbose message would be more clear, like "volume is resyncing changes from ... to ..."